lms/static/js/edxnotes/views/shim.js

```diff
@@ -58,6 +58,31 @@ define([
         return (timeToExpiry > 0) ? timeToExpiry : 0;
     };
 
+    Annotator.Plugin.Tags.prototype.pluginInit = _.compose(
+        function() {
+            // Add screen reader label for edit mode.
+            var index = this.annotator.editor.fields.length -1;
+            var tagsField = 'annotator-field-' + index;
+            $('<label class="sr" for='+ tagsField +'>Edit tags</label>').insertBefore(
```

Comment:
The screen reader label for the text field also is not internationalized, so I followed that pattern (and I'm not sure how the internationalization of the Annotator code works-- there is a "_t" function being used, but I don't know if it maps through to our translation files).

@cptvitamin are we supposed to internationalize text that is only supplied to screen readers?